examples/main.rs

```diff
@@ -17,22 +17,25 @@ fn main() {
         print!("Layer \"{}\":\n\t", layer.data().name);
 
         match layer.layer_type() {
-            tiled::LayerType::TileLayer(layer) => match layer.data() {
-                tiled::TileLayerData::Finite(data) => println!(
+            tiled::LayerType::TileLayer(layer) => match layer {
+                tiled::TileLayer::Finite(data) => println!(
                     "Finite tile layer with width = {} and height = {}; ID of tile @ (0,0): {}",
-                    data.width(),
-                    data.height(),
-                    layer.get_tile(0, 0).unwrap().id
+                    data.data().width(),
+                    data.data().height(),
+                    data.get_tile(0, 0).unwrap().id
                 ),
-                tiled::TileLayerData::Infinite(data) => {
+                tiled::TileLayer::Infinite(data) => {
                     // This is prone to change! Infinite layers will be refactored before 0.10.0
                     // releases.
-                    println!("Infinite tile layer with {} chunks", data.chunks.len())
+                    println!(
+                        "Infinite tile layer; Tile @ (-5, 0) = {:?}",
+                        data.get_tile(-5, 0)
+                    )
                 }
             },
```

Comment:
If we're removing apparently unnecessary commas here, let's also remove this one. :-)

Reply:
If I accidentally removed a comma, that was probably done by rustfmt; Details like that aren't really that important and don't create any issues for the parser/compiler in general.